service/RTC/RTCEvents.js

```diff
@@ -14,6 +14,8 @@ const RTCEvents = {
     DOMINANT_SPEAKER_CHANGED: 'rtc.dominant_speaker_changed',
     LASTN_ENDPOINT_CHANGED: 'rtc.lastn_endpoint_changed',
 
+    SELECTED_UPDATE: 'rtc.selected_update',
```

Comment:
majority of the RTC events use CHANGED word instead of UPDATED. Does it make sense to name it SELECTED_CHANGED ? or IS_SELECTED_CHANGED ?

Reply:
fixed